code/xcvm/cosmwasm/contracts/gateway/clippy.toml

```diff
@@ -0,0 +1,11 @@
+disallowed-types = [
+    # { path = "usize", reason = "variable size" }, # cannot on now, because serde, even if there is no usize in type
```

Comment:
I don’t even see how usize could be disallowed since that’s what Rust uses for lengths.

Reply:
yes, but seems when serde and serde_json operate, they have some switch, which touches usize. 

so not usize bad for serde/json, but usize in api.

Reply:
so tbh half of cases when usize is used can use more concrete size

Reply:
just left comment for people

Reply:
it will forbid usage of usize to cast or declare it, but .len() will work. it is stupid macro on ident declaration, not really parsing type tree seem